playkit/src/main/java/com/kaltura/playkit/plugins/KalturaLiveStatsPlugin.java

```diff
@@ -169,7 +172,7 @@ public void onEvent(PKEvent event) {
     public void onEvent(PlayerEvent.StateChanged event) {
         switch (event.newState) {
             case READY:
-                startTimerInterval();
+                //startTimerInterval();
```

Comment:
this is double call since already started before then in ready it was done again